test/console_callback.erl

```diff
@@ -76,29 +76,28 @@ handle('GET', [<<"api">>, <<"router">>, <<"devices">>], Req, Args) ->
                         }
                     ],
                     undefined};
-            [{devices, {Qs, []}}] ->
-                true = ets:delete(Tab, devices),
+            [{devices, {Qs, [], Refill}}] ->
+                true = ets:insert(Tab, {devices, Refill}),
```

Comment:
A `delete` becomes an `insert`...?

Would you add a bit more context to the PR's description regarding nature of these changes, please?

Reply:
I put those sorts of explanations in commit messages.

> Some tests (xor in particular) rely on being to treat Console endpoints
like a stable store. When we're paginating the results put all the
devices back so we can do it again.